ui/perfherder/compare/ComparePageTitle.jsx

```diff
@@ -77,52 +80,62 @@ export default class ComparePageTitle extends React.Component {
   };
 
   render() {
-    const { inEditMode, pageTitle, newPageTitle } = this.state;
-
-    return !inEditMode ? (
-      <Button
-        className="text-center"
-        size="lg"
-        color="white"
-        onClick={this.goToEditMode}
-        title="Click to change the page title"
-      >
-        <h1 className="page-title-text">
-          {pageTitle}
-          <FontAwesomeIcon
-            icon={faEdit}
-            className="fa-xs align-top edit-icon"
-          />
-        </h1>
-      </Button>
-    ) : (
-      <InputGroup>
-        <Input
-          className="pb-1 col-sm-12 page-title-input"
-          ref={this.inputRef}
-          color="white"
-          style={{
-            textAlign: 'center',
-            fontSize: 'xx-large',
-          }}
-          value={newPageTitle}
-          onChange={event => this.editpageTitle(event.target.value)}
-          onKeyDown={event => this.userActionListener(event)}
-          autoFocus
-        />
-        <Button
-          className="ml-3 my-2"
-          vertical="center"
-          size="lg"
-          color="secondary"
-          onClick={this.injectEnter}
-        >
-          Save
-        </Button>
-        <Button size="lg" color="link" onClick={this.injectEscape}>
-          Cancel
-        </Button>
-      </InputGroup>
+    const { inEditMode, pageTitle, newPageTitle, tabTitle } = this.state;
+
+    return (
+      <React.Fragment>
+        {tabTitle !== '' && (
+          <Helmet>
+            <meta charSet="utf-8" />
+            <title>{tabTitle}</title>
```

Comment:
Replace `{tabTitle}` with something like `{tabTitle || this.props.defaultPageTitle}`.